src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$primary-color: rgba(0, 172, 220, 1);
+$main-accent: rgba(6, 11, 53, 1);
+$secondary: rgba(97, 96, 112, 1);
+$blue-accent: rgba(0, 172, 220, 1);
```

Comment:
The `$primary-color` and `$blue-accent` variables have the same RGBA value. Consider using a single variable to avoid duplication and make future changes easier.